students/dowoon/users/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from users.validators import validate_email, validate_password, validate_duplicate
+
+
+class UserView(View):
```

Comment:
`View`는 기능별로 작성하게 됩니다. `UserView`라고 한다면 너무 많은 의미를 포함하고 있는 것 같습니다. 해당 기능에 맞게 `SignUpView` 로 수정해주세요
```suggestion
class SignUpView(View):
```

Reply:
확인 / 수정했습니다